package.json

```diff
@@ -173,6 +174,7 @@
     "pretty-quick": "1.8.0",
     "prop-types": "15.6.2",
     "react-test-renderer": "16.5.0",
+    "schedule": "0.4.0",
```

Comment:
@drewvolz What made us need this package? I don't recall reading about it being required…

Reply:
It seems to be an internal dep of react. This may be my local environment polluting; I'm going to upgrade to the latest and see if that changes anything.

Reply:
React and RN are tied closely together; you ~~can't~~ shouldn't upgrade one without the other.

Reply:
RN 0.57.3 wants `react@16.6.0-alpha.8af6728`. 

(Also, RN 0.57.4 will be out in ~1 or 2 weeks, and will either go with 16.6.0-final or downgrade to 16.5.0.)

Reply:
I get no warnings from `yarn` when I remove it locally.

Reply:
@hawkrives 

1. `schedule@^0.3.0` is already included as it is a dependency of `react-test-renderer@16.5.0` ([see the yarn lockfile](https://github.com/StoDevX/AAO-React-Native/blob/17c6bcf185f0951cdc493f20efc17721c3884376/yarn.lock#L7135))
2. I've tried adding `schedule@0.4.0` due to advice in [here](https://github.com/facebook/react-native/issues/21150)